modules/models/greeting_models.py

```diff
@@ -0,0 +1,29 @@
+from pydantic import BaseModel, Field
+
+
+class UserInfo(BaseModel):
+    id: str = Field(
+        ..., example="U02RK2AL5LZ", description="The Slack ID of the new user"
+    )
+    name: str = Field(
+        ..., example="julio123", description="The Slack name of the new user"
+    )
+    first_name: str = Field(
+        None, example="Julio", description="The first name of the new user"
+    )
+    last_name: str = Field(
+        None, example="Mendez", description="The last name of the new user"
+    )
+    display_name: str = Field(
+        None, example="julio123", description="The display name chosen by the user"
+    )
+    real_name: str = Field(
+        None,
+        example="Julio Mendez",
```

Comment:
Is Julio Mendez a real person? Normally I see Alice and Bob in stuff like this, but I love the diversity 😄

Reply:
Nah not a real person, I just prefer using something different 😄